### PR TITLE
fix(uploadManuscript): only log error message on science beam failure

### PR DESCRIPTION
**Pull request by dryhten**
re #703

### CODE REVIEW

File: server/xpub/entities/manuscript/resolvers.js [hunk 1]
```diff
@@ -214,7 +214,7 @@ ${err}`,
         }
       } catch (error) {
         logger.warn('Manuscript conversion failed', {
-          error,
+          error: error.message,
```
Comment (diversemix) [inline server/xpub/entities/manuscript/resolvers.js]: So I am guessing that `error` had the entire file contents in as well?

Comment (diversemix) [reply server/xpub/entities/manuscript/resolvers.js]: Are there any other members of `error` that might be useful, other than `message`?

**Comment by dryhten** [comment]
Just tested this on staging environment and it solved https://github.com/elifesciences/elife-xpub/issues/703 but there still seems to be a memory leak on file upload. After the upload is done the memory doesn't go down by the size of the file, so if you keep uploading files the node process will use more and more memory. This needs to be tested (maybe there is a garbage collecting mechanism that will eventually take care of it). I will open an issue for it.

**Comment by diversemix** [comment]
I'm not sure you would expect the memory to go down, this is normally considered wasteful. Then again I'm not sure how the garbage collector works in V8

File: server/xpub/entities/manuscript/resolvers.js (hunk 1 above)
Comment (dryhten) [reply server/xpub/entities/manuscript/resolvers.js]: I think the message is the most relevant. There is also
name: the name of the error (e.g. `StatusCodeError`)
status code: e.g. 500 (this is included in `error.message` as well)
error: the error generated (this is included in `error.message` along with the code)
options: the options/arguments for `request.post`
constructor: the constructor for the error object (the actual code)
response: the response object from `request.post` (including the file)

Comment (diversemix) [reply server/xpub/entities/manuscript/resolvers.js]: Let's leave it as-is, happy for it to get merged.

**Pull request merged**